booster/library/Booster/Pattern/ApplyEquations.hs

```diff
@@ -331,6 +335,10 @@ iterateEquations direction preference startTerm = do
             config <- getConfig
             currentCount <- countSteps
             when (coerce currentCount > config.maxIterations) $ do
+                currentRecursionDepth <- getRecusionDepth
+                -- FIXME if this exception is caught in evaluatePattern',
+                --       then CtxAbort is a wrong context for it.
+                --       We should emit this log  entry somewhere else.
                 withContext CtxAbort $ do
                     logWarn $
                         renderOneLineText $
```

Comment:
We could probably log this  in `evaluatePattern'` where we decide whether or not to use the final term.
The warning will still show up for recursive evaluations where we catch the exception, too. Maybe not useless to notify the user about it, though? (so we could just use `CtxWarn`).

Reply:
I've removed `CtxAbort`, so these will show up as warnings now.